faq/sql-faq.md

```diff
@@ -11,6 +11,16 @@ This document summarizes the FAQs related to SQL operations in TiDB.
 
 See [System Variables](/system-variables.md).
 
+## The order of records for a SELECT statement without ORDER BY clause is not as same as MySQL.
+
+It's not a bug. The default order of records depends on various situation without any guarantee of consistency. 
+
+The relevant text in the [SQL-92](http://www.contrib.andrew.cmu.edu/~shadow/sql/sql1992.txt) spec is
```

Comment:
```suggestion
## The order of results is different from MySQL

Unless the query includes `ORDER BY` on a unique column (or set of columns), TiDB is not guaranteed to return results in the same order as MySQL.

This is legal according to SQL standards. The relevant text in the [SQL-92](http://www.contrib.andrew.cmu.edu/~shadow/sql/sql1992.txt) spec is
```

Reply:
Technically it could still be ambiguous if there is an order by (non-distinct). I updated the text to handle this case.

Reply:
In fact, those are different aspects
1) Without `ORDER BY`, the order of a `SELECT` statement's results is inconsistent. This is what I want to explain in this FAQ
2) With an `ORDER BY`, the columns out of `ORDER BY`'s scops is inconsistent.

Case 2) could be an extra section to describe.

Reply:
I've updated my description to explain these two aspects separately. PTAL again.